styles.css

```diff
@@ -7,24 +7,70 @@ body {
 	margin: 0;
 }
 
-/* @font-face {
-	font-family: 'Montserrat';
-	src: url("https://fonts.googleapis.com/css2?family=Montserrat&display=swap");
-} */
-
 header {
 	background-image: url("images/background-chillis.jpg");
 	background-size: cover;
 	background-repeat: no-repeat;
 	width: 100%;
-	height: 600px;
-	color: white;
+	height: 1000px;
+	color: #CBA135;
+	font-family: 'Prata', serif;
+	display: flex;
+	align-items: center;
+	flex-direction: column;
+}
+
+header h1 {
+	font-size: 10rem;
+	border-bottom: 10px;
+	margin-top: 30px;
 }
 
 main {
 	background-color: black;
 	color: white;
 	display: flex;
-	justify-content: center;
+	align-items: center;
+	flex-direction: column;
+}
+
+main h1 {
+	color: white;
+	font-family: 'Prata', serif;
+	font-size: 5rem;
+	border-bottom: 10px;
+	margin-top: 30px;
 }
 
+table {
+	color: white;
```

Comment:
might be wrong but i dont think this is needed as it is already white from main?